apm-hub-helm/values.yaml

```diff
@@ -0,0 +1,99 @@
+# Default values for apm-hub.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+deployment:
+  label: apm-hub
+
+image:
+  repository: docker.io/flanksource/apm-hub
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: "apm-hub"
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: "apm-hub-sa"
```

Comment:
remove - serviceAccount must always be created with a non-configurable name

Reply:
Reduced the configurability of serviceAccount. Can only add Annotations now.